backend/corpora/dataset_submissions/requirements.txt

```diff
@@ -0,0 +1,10 @@
+# TODO: cleanup after authz refactor
```

Comment:
We won't need most of these dependencies after merging in Trent's refactoring of the auth code.